.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -237,7 +237,9 @@
       git config core.fileMode false  # used to tell git not to identify permission changes on files as changes
       chmod -R 777 ./Packs  # required for the lint, we use the permissions of the file when running pytest within the containers
       echo "the Packs changes between upload commit ${LAST_UPLOAD_COMMIT} to master commit ${CI_COMMIT_SHA} is:"
-      git status -- Packs  # show the differences between the upload commit to the master branch for the Packs folder
+      git status -- Packs  # show the differences between the upload commit to the master branch for the Packs folder before commit
+      git commit -am "updated branch to upload commit ${LAST_UPLOAD_COMMIT}"
```

Comment:
maybe its better to commit only the `Packs` folder and not all the changed files.